arelle/plugin/validate/DBA/DisclosureSystems.py

```diff
@@ -10,3 +24,7 @@
 MULTI_TARGET_DISCLOSURE_SYSTEMS = [
     ARL_2024_MULTI_TARGET_PREVIEW
 ]
+
+DKFIN_MULTI_TARGET_DISCLOSURE_SYSTEMS = [
+    DKFIN_2024_MULTI_TARGET_PREVIEW
+]
```

Comment:
The name `MULTI_TARGET_DISCLOSURE_SYSTEMS` no longer makes sense if it doesn't include the new dkfin multi-target disclosure system. Can you rename it to `ARL_MULTI_TARGET_DISCLOSURE_SYSTEMS`?